hedera/deploy-its.js

```diff
@@ -0,0 +1,600 @@
+const { ethers } = require('hardhat');
+const {
+    Wallet,
+    Contract,
+    getDefaultProvider,
+    utils: { defaultAbiCoder, isAddress },
+} = ethers;
+
+const {
+    deployContract,
+    printWalletInfo,
+    saveConfig,
+    printInfo,
+    printWarn,
+    printError,
+    mainProcessor,
+    prompt,
+    sleep,
+    getBytecodeHash,
+    getGasOptions,
+    isContract,
+    isValidAddress,
+    getDeployOptions,
+    getDeployedAddress,
+    wasEventEmitted,
+} = require('../evm/utils');
+
+const { getContractJSONWithHTS } = require('./utils.js');
+const { printHederaNetwork, addBaseOptions } = require('./cli-utils.js');
+
+const { addEvmOptions } = require('../evm/cli-utils');
+const { Command, Option } = require('commander');
+const { WHBAR_ABI, fundWithWHBAR } = require('./fund-whbar.js');
+const { DEFAULT_TOKEN_CREATION_PRICE_TINY_CENTS, TINY_PARTS_PER_WHOLE } = require('./token-creation-price.js');
+
+/**
+ * Function that handles the ITS deployment.
+ * @param {*} wallet
+ * @param {*} chain
+ * @param {*} deployOptions
+ * @param {*} operatorAddress
+ * @param {*} skipExisting
+ * @param {*} verifyOptions
+ */
+
+async function deployAll(config, wallet, chain, options) {
```

Comment:
Hmm, not a fan of duplicating the whole script, it's gonna lead to out-of-sync changes and critical errors as a result. Why can't we just make changes to `evm/deploy-its.js` with the appropriate helpers to support Hedera deployment (similar to what was done for Hyperliquid)?